### DISCUSSION TITLE
Is possible to get the headers of getReport()

### DISCUSSION THREAD

**Discussion by set92** [pitchmuc/adobe_analytics_api_2.0#75] [Q&A] [answered]
With getReport() you get the results of the table, but in Adobe Analytics there is a header that tends to show the mean of the data.

For example, in this video https://youtu.be/6SSR4rBStZ8?t=1157, the header is Marketing Cloud ID (v6), and the value 550. I didn't find a parameter to also get that value, so not sure if it is possible to return it.

Edit: I just tried with summaryData, but is not that value. The headers are unique per day, and the mean on summaryData is with the data that returns the call to getReport()

**Comment by pitchmuc** [comment, marked as answer]
Hello @set92 
This is planned in the future release. 
A new method will be available: `getReport2` that will return more information from your request. 
I still need to document that and release it. 
Stay tune in the next week(s)

**Reply by set92** [reply]
Thanks, saw that method in the source code, I tried to call it, but I couldn't, and I thought it was an old method :D.

**Reply by pitchmuc** [reply]
It is not officially release so if you are using the 0.3.0.post2 (0.3.0) version, you won't have this method available yet. 
You can try to build the library from the github if you want to use it before the release.

**Comment by set92** [comment]
If it helps as betatesting, I downloaded the package from GitHub (v0.3.1) and tried to make it work, but my JSON doesn't have statistics (I think is safe to show you if it helps you to debug something), but seems that the code always expects a statistics section.

And after I added some random statistics with :
```
,
    "statistics": {
        "functions": [
            "col-max",
            "col-min"
        ]
    }
```
It gave another error `AttributeError: 'Analytics' object has no attribute 'getFilter'` raising from `seg = self.analyticsObject.getFilter(filter["segmentId"])` which, I think, is because I have to initialize the library differently than before. 

Edit: Right now I'm initializing the library as:
```python
import aanalytics2 as api2

api2.importConfigFile('config_analytics_jwt.json')
login = api2.Login()
cids = login.getCompanyId()
cid = cids[0]['globalCompanyId']
company = login.createAnalyticsConnection(cid)
```

**Comment by pitchmuc** [comment]
Good catch ! It comes from a cjapy library reference (as segments are called filters there). 
I just uploaded a fix in the repo now. 
I am still working on it at the moment
